code/game/gamemodes/steal_items.dm

```diff
@@ -186,6 +186,21 @@
 	typepath = /obj/item/weapon/switchtool/holo
 	protected_jobs = list("Research Director")
 
+/datum/theft_objective/traitor/clownPDA
+	name = "the clown's PDA"
+	typepath = /obj/item/device/pda/clown //A printed PDA painted to look like a clown PDA is just device/pda, so no cheating
+	protected_jobs = list("Clown")
+
+/datum/theft_objective/traitor/telesci
+	name = "the telescience computer's circuit"
+	typepath = /obj/item/weapon/circuitboard/telesci_computer
+	protected_jobs = list("Research Director", "Scientist", "Roboticist")
```

Comment:
Geneticist and Mechanic also have access to this room on Box, Meta, and Packed.
Paramedic has access to this room on Deff and Roid.

Reply:
I was considering the geneticist, but I thought since seeing a geneticist in sci is a massive rarity it was fine to leave them out.
Definitely adding mechanics and parameds though